graphql_api/tests/test_repository.py

```diff
@@ -893,14 +892,15 @@ def test_test_results_no_tests(self) -> None:
     def test_branch_filter_on_test_results(self) -> None:
```

Comment:
am I missing something or are there no assertions on this test and the next two?

Reply:
i think the github file viewer is just not showing them, they are there

Reply:
oh weird, yeah I see them now